### PR TITLE
Remove unnecessary `UpdatePlistPreprocessor` aggregate target and build phase

### PR DESCRIPTION
**Pull request by mokagio**
We haven't been using the credentials this script used as input for a long time, as shown by the warning "Google Client ID not found".

image

## Testing

As mentioned, this build phase has been essentially a no-op for a while. Just in case, I installed the one-off build via App Center to verify I could still login and interact with WordPress. I posted pbMe7B-1c-p2 from my main account and pbMe7B-1f-p2 from my test account which uses Google login.

## Regression Notes
1. Potential unintended areas of impact. N.A.
2. What I did to test those areas of impact (or what existing automated tests I relied on). N.A.
3. What automated tests I added (or what prevented me from doing so). N.A.

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/WordPress.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -17707,21 +17675,6 @@
...
-			files = (
-			);
-			inputPaths = (
-			);
-			name = "Update Plist Preprocessor file";
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "[ -a $WPCOM_CONFIG ] && source $WPCOM_CONFIG\n touch Info.plist\n echo \"\" > InfoPlist.h\n echo \"\" > InfoPlist-alpha.h\n echo \"\" > InfoPlist-internal.h\n if [ \"x$GOOGLE_LOGIN_SCHEME_ID\" != \"x\" ]; then\n echo \"#define GOOGLE_LOGIN_SCHEME_ID $GOOGLE_LOGIN_SCHEME_ID\" >> InfoPlist.h\n echo \"#define GOOGLE_LOGIN_SCHEME_ID $GOOGLE_LOGIN_SCHEME_ID\" >> InfoPlist-alpha.h\n echo \"#define GOOGLE_LOGIN_SCHEME_ID $GOOGLE_LOGIN_SCHEME_ID\" >> InfoPlist-internal.h\n else\n echo \"warning: Google Client ID not found\"\n fi\n";
```
Comment (mokagio) [inline WordPress/WordPress.xcodeproj/project.pbxproj:17722]: At the end of the line (_Xcode Y U NO break scripts to multiple lines?!_) you can see the `echo` that generates the warning mentioned in the PR description:

image

File: .gitignore [hunk 2]
```diff
@@ -103,3 +100,12 @@ WordPress/Credentials/Secrets.swift
...
 WordPress/Credentials/wpcom_internal_app_credentials
+# These headers, too, are not longer used or generated by the build process,
+# but we wnt to keep ignoring them to avoid them being accidentally committed.
+#
+# We shall remove this once enough time has passed since they became unused.
+# Maybe a couple of years would be enough for most developers to change machine
+# and no longer have these files in their local checkout.
+WordPress/InfoPlist.h
+WordPress/InfoPlist-alpha.h
+WordPress/InfoPlist-internal.h
```
Comment (mokagio) [inline .gitignore:111]: If you check your local copy, you'll likely have these files and they'll be empty.

**Review by oguzkocer — APPROVED** [review]
I am reluctantly approving this as the changes accompanied by the explanations make sense to me. However, you might want to get another 👍 from an iOS infrastructure engineer - I'll leave that decision to you and rest of the team.

**Comment by mokagio** [comment]
> you might want to get another 👍 from an iOS infrastructure engineer

I posted about this on Slack at the time of Oguz's comment, but it got lost.

Explicit ping to @leandroalonso, @momo-ozawa, @hassaanelgarem since you folks will be working on warnings soon and @jkmassel as someone very knowledgeable in this area. Thanks! 🙇‍♂️

**Review by momo-ozawa — APPROVED** [review]
Tested Google sign in, works as expected

**Comment by mokagio** [comment]
Thank you for following up on this @momo-ozawa 🙇‍♂️

**Pull request merged**